http/HttpCache.cc

```diff
@@ -85,6 +85,22 @@ namespace http {
         return size_in_megabytes;
     }
 
+    unsigned long long HttpCache::getCacheExpiresTime() {
+        bool found = false;
+        string time;
+        unsigned long time_in_seconds = 0;
+        TheBESKeys::TheKeys()->get_value(HTTP_CACHE_EXPIRES_TIME_KEY, time, found);
+
+        if (found) {
+            std::istringstream iss(time);
+            iss >> time_in_seconds;
+        } else {
+            time_in_seconds = 3600;
```

Comment:
Use a constant like 'REMOTE_RESOURCE_DEFAULT_EXPIRED_INTERVAL' here instead of 3600.